### ISSUE TITLE
Embedded images not displaying

### ISSUE THREAD

**Issue by cbdabner** [djfarrelly/MailDev#69]
Images embedded as base64 with Content-IDs, e.g: `<img src="cid:{Content-ID}"/>` are displaying as attachments but fail with net::ERR_UNKNOWN_URL_SCHEME in **HTML** view.

If you look at the **Source** view, you can see something along the lines of:

```
----boundary_1_...
Content-Type: image/png
Content-Transfer-Encoding: base64
Content-ID: <effcd49f6cec4e688d11ddd49cd6e9c5>

{Base64-Data}...

----boundary_1_...--
```

So maybe you can lookup the data related to the Content-ID in the img src, and in the html view replace the image with: `<img src="data:{Content-Type};{Content-Transfer-Encoding},{Base64-Data}"/>`?

**Comment by djfarrelly** [comment]
hey @cbdabner, do you happen to have a copy of a full email source I could use to write a test and a fix for? Would this example source work for testing? http://stackoverflow.com/a/9110164/1704830

**Comment by cbdabner** [comment]
That source looks pretty reasonable, but I'll provide a complete copy tomorrow as received by MailDev.

**Comment by cbdabner** [comment]
I've uploaded an example of one as a gist:
https://gist.github.com/cbdabner/2aa6b06f9dc0b2c41278#file-maildev_issue69

**Comment by djfarrelly** [comment]
Fixed with version 0.11.0

**Comment by bitclaw** [comment]
This would be great to see fixed :dancers: ! Currently getting the same error as well:

![maildev_cid_images_issue_blur](https://cloud.githubusercontent.com/assets/2736635/10963850/590a8fcc-8371-11e5-91e6-74d8937fc6be.png)
